src/library/dao/playlistdao.cpp

```diff
@@ -757,6 +757,10 @@ void PlaylistDAO::removeTracksFromPlaylists(const QList<TrackId>& trackIds) {
                 it != playlistsTrackIsInCopy.constEnd(); ++it) {
             if (it.key() == trackId) {
                 const auto playlistId = it.value();
+                // keep tracks in history playlusts
```

Comment:
typo

Reply:
The function is called removeTracksFromPlaylists() without any exceptions. Now it contains some special logic in the implementation that is not mentioned anywhere. Functions should do what they are promising.

Reply:
Well, if it was removeTracksFrom**All**Playlists() I'd _fully_ agree.
What do you propose?
We could simply name it removeTracksFromPlaylistsExceptHistory() since we agreed to keep all tracks in history playlists, no matter if tracks are purged or hidden.

Reply:
ping @uklotzde